src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc) format('truetype');
+}
+
+body {
+  margin: 0;
+}
+
+.search__form {
+  display: block;
+  font-family: Avenir, serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__big {
+  margin: 20px 8px;
+  height: 70px;
+}
+
+.search__big :first-child {
```

Comment:
The ':first-child' selector is used here, but it may not select the intended element if the input is not the first child of its parent. Consider using a more specific class or ID selector to target the input elements directly.